mesh/accounts/services.py

```diff
@@ -0,0 +1,80 @@
+from .models import Account
+from django.http import JsonResponse
+from django.core.mail import send_mail
+import pyotp, bcrypt, json, os
+
+def decrypt(password, salt):
+    pepper = os.getenv("PEPPER")
+    password = f"{password}{pepper}".encode('utf-8')
+    return bcrypt.hashpw(password,salt)
+
+def getUserServices(request):
```

Comment:
This is somewhat nitpicky, but python functions should be named with snake_case. This would also be good so that this code can be consistent with the rest of the codebase. Another small thing would be to use a formatter like autopep8 or black, though since most of the codebase doesn't have some formatter set in place that isn't as much of an issue for this PR specifically (though it would be nice to try and set that standard now!)

Reply:
I reformatted using black and also renamed the functions to snake_case c6522ce

Reply:
👍🏼